corehq/ex-submodules/dimagi/utils/couch/undo.py

```diff
@@ -72,6 +72,19 @@ def undo_delete(document, save=True):
 
 
 def remove_deleted_doc_type_suffix(doc_type):
```

Comment:
This already has tests: https://github.com/dimagi/commcare-hq/blob/a49dc7ed67b67dd9581701dff742487fbf54cb77/corehq/apps/userreports/tests/test_utils.py#L89

Though I don't know why I put it in that app. We could move it this app.